spec/values/ParticipleSpec.js

```diff
@@ -0,0 +1,36 @@
+var Participle = require( "../../js/values/Participle.js" );
+
+describe( "A test for checking the Participle", function() {
+	it( "checks the properties of a participle object without a passive", function() {
+		var mockParticiple = new Participle( "geschlossen", "Es wird geschlossen worden sein.", [ "wird", "worden" ],  "irregular" );
+		expect( mockParticiple.getParticiple() ).toBe( "geschlossen" );
+		expect( mockParticiple.getSentencePart() ).toBe( "Es wird geschlossen worden sein." );
+		expect( mockParticiple.getType() ).toBe( "irregular" );
+		expect( mockParticiple.getAuxiliaries() ).toEqual( [ "wird", "worden" ]);
+	});
+
+	it( "checks the properties of a participle object without a passive", function() {
+		var mockParticiple = new Participle( "geschlossen", "Wir werden geschlossen haben.",  [ "werden" ],  "irregular" );
```

Comment:
What do you expect the function to do with weird symbols like quotes, ampersands etc..